lib/ansible/module_utils/docker/common.py

```diff
@@ -423,7 +425,7 @@ def _get_connect_params(self):
     def _handle_ssl_error(self, error):
         match = re.match(r"hostname.*doesn\'t match (\'.*\')", str(error))
         if match:
-            self.fail("You asked for verification that Docker host name matches %s. The actual hostname is %s. "
+            self.fail("You asked for verification that Docker daemon host name matches %s. The actual hostname is %s. "
```

Comment:
I am not sure if this won't be confusing. Docker host is the host where you perform the operations. But you may still have a daemon running on the host where you run playbook but execute tasks on other hosts. I would leave it as it is now

Reply:
This is talking about the hostname returned by the docker daemon (i.e. its TLS certificate), not about actual hostnames. But yeah, this message is not very clear.

Reply:
I've tried to improve the message in 506ee30c9f08e23fac32ac2971e4a33604a9b98b.